lib/elixir/lib/enum.ex

```diff
@@ -1341,8 +1341,8 @@ defmodule Enum do
   If multiple elements are considered maximal, the first one that was found
   is returned.
 
-  Calls the provided `empty_fallback` function and returns its value if
-  `enumerable` is empty. The default `empty_fallback` raises `Enum.EmptyError`.
+  Calls the provided `empty_fallback/0` function and returns its value if
```

Comment:
alright. another good thing to do in terms of clarifying that we are dealing with an anonymous function, is to mention the word  provided as in "provided `foo/2` function", this example does that, but others int he docs don't. ....so I will do that.